src/t8_cmesh/t8_cmesh_readmshfile.cxx

```diff
@@ -954,9 +955,12 @@ t8_cmesh_msh_file_4_read_eles (t8_cmesh_t cmesh, FILE *fp,
         /* *INDENT-OFF* */
 
         /* Calculate the parametric geometries of the tree */
-        if (occ_geometry != NULL)
+        if (use_occ_geometry)
         {
 #if T8_WITH_OCC
+          const t8_geometry_c *occ_geometry_base = t8_cmesh_get_tree_geometry (cmesh, tree_count);
```

Comment:
When running the naca tutorial in debug mode, I get an assertion here: `Abort: Assertion 't8_cmesh_is_committed (cmesh)'`.
Moreover, the generated forests from the release build are deformed. This may be connected somehow.
![grafik](https://user-images.githubusercontent.com/49643115/207326726-6461a188-0dba-44ab-ad0c-01fb6f3dfe24.png)

Reply:
that does not look good.
Can you post the exact call that you are executing and the input file?

Reply:
`mpirun -np 8 ./t8_features_curved_meshes -p -f source/naca6412 -o`
[naca6412.zip](https://github.com/DLR-AMR/t8code/files/10259111/naca6412.zip)

Reply:
![image](https://user-images.githubusercontent.com/7125317/213227551-e5ff9db5-0f70-4590-bd8d-76b5da6c74e5.png)

The same run with the new code :)

Reply:
Colors are processes